apps/build-onchain-apps/app/buy-me-coffee/page.tsx

```diff
@@ -0,0 +1,13 @@
+import { generateMetadata } from '../../src/utils/generateMetadata';
+import BuyMeCoffeePage from '.';
```

Comment:
Page from index? 🤔 is it how the cool kids to this these days?

Reply:
Ya so the page is like the server component, and then you import the client component into it
https://nextjs.org/docs/pages/building-your-application/upgrading/app-router-migration#step-4-migrating-pages